pkg/skaffold/schema/v1alpha2/config.go

```diff
@@ -226,6 +226,10 @@ func (c *SkaffoldConfig) ApplyProfiles(profiles []string) error {
 
 	c.Profiles = nil
 
+	// lets populate any missing default values
+	setDefaultDockerfiles(c)
```

Comment:
There's more default values set when we read the configuration. Shouldn't we try to reuse the same code to avoid duplication and missing things?

Reply:
sounds good to me. I was just trying to un-break profiles